daemon/inertiad/project/data.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"errors"
 
+	gcrypto "crypto/rand"
```

Comment:
might want to leave this as `rand` - there's no need to rename it
```suggestion
	"crypto/rand"
```